autoconfigure-storage/src/main/java/com/google/cloud/trace/zipkin/autoconfigure/ZipkinStackdriverStorageProperties.java

```diff
@@ -16,26 +16,36 @@
 
 package com.google.cloud.trace.zipkin.autoconfigure;
 
+import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @ConfigurationProperties("zipkin.storage.stackdriver")
+@Data
 public class ZipkinStackdriverStorageProperties {
   private String projectId;
   private String apiHost = "cloudtrace.googleapis.com";
+  private Executor executor = new Executor();
```

Comment:
here's an example of how to test this

https://github.com/openzipkin/zipkin/pull/1450/files#diff-d5ed7e4d8a81b9885985b99989e2a628R93

conventionally, snake case is preferred with properties.

the main server readme has a bit on how to set things like this (which is a summary of spring boot docs)
https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md#configuration-for-the-ui